dashboard/apps/dashboard/spreadsheets.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+from datetime import date
+import re
+
+from openpyxl.styles import Style, Font
+from openpyxl.workbook import Workbook
+
+from dashboard.libs.date_tools import parse_date
+
+
+class Products:
```

Comment:
Nice quote from Zen of Python. I'm not sure if it applies to this though. For Python3, I tend to not bother with `(object)` as the standard docs all seem to go this way.